src/savegame_json.cpp

```diff
@@ -3996,7 +3996,7 @@ void submap::load( JsonIn &jsin, const std::string &member_name, int version )
             int rad_num = jsin.get_int();
             for( int i = 0; i < rad_num; ++i ) {
                 if( rad_cell < SEEX * SEEY ) {
-                    set_radiation( { 0 % SEEX, rad_cell / SEEX }, rad_strength );
+                    set_radiation( { rad_cell % SEEX, rad_cell / SEEX }, rad_strength );
```

Comment:
I wonder why operators are different too.

Reply:
It should be 1D to 2D conversion.